site/_data/v0-14-3toc.yml

```diff
@@ -0,0 +1,28 @@
+toc:
+  - title: Setting up
+    subfolderitems:
+      - page: Getting started
+        url: /index.html
+      - page: Config
+        url: /sonobuoy-config
+      - page: Snapshot layout
+        url: /snapshot
+      - page: ImagePullSecrects
+        url: /pullsecrets
+  - title: Use cases
+    subfolderitems:
+      - page: Conformance testing
+        url: /conformance-testing
+      - page: Custom Registries & Airgap Testing
+        url: /airgap
+      - page: Plugins
+        url: /plugins
+  - title: Next steps
+    subfolderitems:
+      - page: Customization
+        url: /gen
+      - page: Make your own plugins
```

Comment:
I think we need to beef up the docs here for the upcoming versions, explaining _how_ to create a new plugin. Then we can also link to the example directory in that text.

Reply:
Yea, I did a blog post last time about this and we need to give stuff like that better visibility for users who start at github (rather than the blog itself)